packages/@webex/plugin-cc/src/services/task/index.ts

```diff
@@ -0,0 +1,83 @@
+import {LocalMicrophoneStream} from '@webex/calling';
+import {getErrorDetails} from '../core/Utils';
+import {LoginOption} from '../../types';
+import {CC_FILE} from '../../constants';
+import routingContact from './contact';
+import {ITask, TaskId, TaskResponse, TaskData} from './types';
+import WebCallingService from '../WebCallingService';
+
+export default class Task implements ITask {
+  private contact: ReturnType<typeof routingContact>;
+  private localAudioStream: LocalMicrophoneStream;
+  private webCallingService: WebCallingService;
+  public data: TaskData;
+
```

Comment:
Should we be sending in the call object here as well since Task and Call are related to each other? How do we know which task corresponds to which call?
What if we have 2-3 calls on an agent at the same time?

Reply:
Actually, IIRC once an agent answers a call they go into a different state so no other call should arrive.. so maybe my scenario won't happen.. but we should try to link the call and the task somehow.

Reply:
There will be one call at a time

Reply:
@Kesari3008 should we still link a call and task since a call is still a task?

Reply:
I have tested this, incoming event is not even received for the second call